sources/Outputs/src/ParGenerator.cpp

```diff
@@ -142,6 +142,10 @@ boost::shared_ptr<parameters::MacroParameterSet> ParGenerator::buildGeneratorMac
     }
   }
 
+  if (!def.hasRpcl() || def.isUsingDiagram()) {
+    macroParameterSet->addParameter(helper::buildParameter("generator_QDeadBand", 0.0001));
+    macroParameterSet->addParameter(helper::buildParameter("generator_UDeadBand", 0.0001));
```

Comment:
Is that a good practice to hard code this values here or shall we add them in the constants ?

Reply:
it is only used in this class so I didn't create a shared constant, but I can if you think it is important